isolate/test/index.ts

```diff
@@ -624,15 +624,15 @@ describe('isolate', function() {
         };
       }
 
-      function MyDataflowComponent(sources: {other: any}) {
+      function MyDataflowComponent(sources: {other: unknown}) {
```

Comment:
Same case as [this comment](https://github.com/cyclejs/cyclejs/pull/873/files#r251171675)